api/handler/get.go

```diff
@@ -168,7 +168,6 @@ func (h *handler) GetObjectHandler(w http.ResponseWriter, r *http.Request) {
 		ObjectInfo: info,
 		Writer:     w,
 		Range:      params,
-		VersionID:  p.VersionID,
```

Comment:
It seems like the field is not used, also we have `versionID` in `ObjectInfo` field